R/stash.R

```diff
@@ -38,7 +38,10 @@ stash <- function(var, code, depends_on = NULL, functional = FALSE, verbose = TR
   if (is.null(var)) stop("`var` cannot be NULL")
   if (formatted_code == "NULL") stop("`code` cannot be NULL")
 
-  new_hash_tbl <- make_hash_table(formatted_code, depends_on)
+  # The environment where all code is evaluated and variables assigned.
+  target_env <- parent.frame()
```

Comment:
Great idea just getting the parent frame at the beginning. Much easier solution than trying to dig through layers of function calls later.